CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
  * `TimelockController`: Add a separate canceller role for the ability to cancel. ([#3165](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/3165))
  * `draft-ERC20Permit`: replace `immutable` with `constant` for `_PERMIT_TYPEHASH` since the `keccak256` of string literals is treated specially and the hash is evaluated at compile time. ([#3196](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/3196))
  * `ERC20Wrapper`: the `decimals()` function now tries to fetch the value from the underlying token instance. If that calls revert, then the default value is used. ([#3259](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/3259))
+ * `MerkleProof`: added `multiProofVerify` to open new airdrop mechanisms and `_hashPair` to process proofs.
```

Comment:
- IMO, the changelog should explain what `multiProofVerify` actually does rather than mentioning possible use-cases
- `_hashPair` is a private function. It should not be documented in the Changelog.
- we need a link to the PR

Reply:
Thank you for clarifying